index.html

```diff
@@ -6395,8 +6394,8 @@ <h2 id="AudioContextInfo">
             </h2>
             <p>
               The <code>AudioContextInfo</code> dictionary provides an
-              <a><code>AudioWorkletGlobalScope</code></a> with a view of an
-              <a><code>AudioContext</code></a>.
+              <a>AudioWorkletGlobalScope</a> with a view of a
```

Comment:
This is a small correction on AudioContextInfo. I am simply changing AudioContext to BaseAudioContext.